packages/guesstimator/formatter/formatters/lib.js

```diff
@@ -57,3 +57,39 @@ export function regexBasedFormatter(re, guesstimateTypeFn = getGuesstimateType,
     _numbers(text) { return _.chunk(text.match(re).slice(1), 2).map(([num, suffix]) => parseNumber(num, suffix)) },
   }
 }
+
+
+
+
+
+// The function: shorthandIntoLognormalFormattingStep 
+// is used in foretold/packages/guesstimator/formatter/formatters/Function.js
```

Comment:
I'm not a fan of comments like this, because:
1. It's easy enough to search the codebase to figure out where it's used.
2. The presence of this implies that if someone were to use this elsewhere, they should also need to add to the comment. This would get pretty messy pretty quickly, and likely get out of sync.